app/javascript/orangelight/availability_show.js

```diff
@@ -20,7 +20,7 @@ export default class AvailabilityShow extends AvailabilityBase {
   constructor() {
     super();
     this.id = '';
-    this.host_id = '';
+    this.host_id = [];
```

Comment:
Could we rename to `host_ids`, since there may be multiple?

Reply:
sure